module/zfs/vdev_raidz.c

```diff
@@ -412,13 +433,18 @@ vdev_raidz_map_alloc(zio_t *zio, uint64_t unit_shift, uint64_t dcols,
 	ASSERT3U(rm->rm_nskip, <=, nparity);
 
 	for (c = 0; c < rm->rm_firstdatacol; c++)
-		rm->rm_col[c].rc_data = zio_buf_alloc(rm->rm_col[c].rc_size);
+		rm->rm_col[c].rc_abd =
+		    abd_alloc_linear(rm->rm_col[c].rc_size, B_FALSE);
```

Comment:
So the illumos patch raidz code was very different from our since we vectorized the raidz code. We did a quick and dirty get it working patch for now just to get something up and running but now we're resolving the technical debt like a proper kmap_atomic kunmap_atomic based map implementation etc.